src/BlazorWorker.ServiceFactory/WorkerBackgroundServiceDependencies.cs

```diff
@@ -1,16 +1,14 @@
-﻿using BlazorWorker.WorkerBackgroundService;
-using BlazorWorker.WorkerCore;
-using System.Linq;
+﻿using System.Linq;
 
 namespace BlazorWorker.BackgroundServiceFactory
 {
     internal class WorkerBackgroundServiceDependencies
     {
         public static string[] BlazorWorkerDependentAssemblyFilenames = new[] {
-            $"{typeof(BaseMessage).Assembly.GetName().Name}.dll",
-            $"{typeof(WorkerInstanceManager).Assembly.GetName().Name}.dll",
+            $"{typeof(Core.IWorker).Assembly.GetName().Name}.dll",
```

Comment:
At some point I guess that `BaseMessage` was moved from `BlazorWorker.Core` to `BlazorWorker.WorkerBackgroundService`. I was occasionally getting errors that the former assembly was not found, even though everything else seemed to work. This error becomes more strange because the latter assembly was not depending on the former, but I find no other explanation to why the same assembly was getting referenced twice.

Regardless, the assembly is now added and the type names were partially qualified to signify the assemblies they came from.